alphagov/paas-cf
# Cflinuxfs4 rollout

What
----

The following changes are included in this PR:
- Introducing [cflinuxfs4-compat-release](https://github.com/cloudfoundry/cflinuxfs4-compat-release) support.
- Upgrade [cf-deployment to v30.5.0](https://github.com/cloudfoundry/cf-deployment/releases/tag/v30.5.0).
- Upgrade [cflinuxfs3 to 0.369.0](https://github.com/cloudfoundry/cflinuxfs3/releases/tag/0.369.0) because of lots of vulnerability fixes.
- Upgrade [ubuntu-jammy buildpack to v1.125](https://github.com/cloudfoundry/bosh-linux-stemcell-builder/releases/tag/ubuntu-jammy%2Fv1.125) as dictated by [cf-deployment v30.5.0](https://github.com/cloudfoundry/cf-deployment/blob/v30.5.0/cf-deployment.yml#L2954).
- Updated [cf-acceptance-tests go dependencies to cf30.5](https://github.com/cloudfoundry/cf-acceptance-tests/tree/cf30.5)

How to review
-------------

- Use the branch to a dev environment and see all concourse jobs go green.
- Verify that there are buildpacks of the same version and specs installed for both stacks cflinuxfs3 and cflinuxfs4.
- Verify that the buildpacks are aligned with the current buildpacks staged in main branch.
- Verify that all pass-cf apps are using the new cflinuxfs4-compat stack.
- Verify that [cf-deployment to v30.5.0](https://github.com/cloudfoundry/cf-deployment/releases/tag/v30.5.0) and [cflinuxfs3 to 0.369.0](https://github.com/cloudfoundry/cflinuxfs3/releases/tag/0.369.0) are indeed used after this change gets applied on the dev environment.

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨
